test/unit/BCoWHelper.tree

```diff
@@ -0,0 +1,28 @@
+BCoWHelperTest::constructor
+└── when called
+    ├── it should set factory
+    └── it should set app data from factory
+
+BCoWHelperTest::tokens
+├── when pool is not registered in factory
+│   └── it should revert
+├── when pool has less than 2 tokens
+│   └── it should revert
+├── when pool has more than 2 tokens
+│   └── it should revert
+├── when pool tokens have different weights
+│   └── it should revert
+└── when pool is supported
+    └── it should return pool tokens
+
+BCoWHelperTest::order
+├── when the pool is not supported
+│   └── it should revert
+├── when the pool is supported
+│   ├── it should query the domain separator from the pool
+│   ├── it should return a valid pool order
+│   ├── it should return a commit pre-interaction
+│   ├── it should return an empty post-interaction
+│   └── it should return a valid signature
+└── given a price vector
```

Comment:
cc-ing this to @drgorillamd, i'm breaking standard with `given` to not-branch but just make a test with a different focus, in this one in particular is fuzzing the balances and price vectors, it could have been done in the same scope as `when the pool is supported`, but think that separation of affairs is good there, since the code is looking at other things, wdyt?

Reply:
`given` ftw :D cc @gas1cent ?

Reply:
OK FINE